docs/NativeModulesIOS.md

```diff
@@ -198,6 +198,7 @@ The native module can signal events to JavaScript without being invoked directly
 JavaScript code can subscribe to these events:
 
 ```javascript
+var DeviceEventEmitter = require('RCTDeviceEventEmitter');
```

Comment:
`DeviceEventEmitter` is part of `react-native` exports now

  var { DeviceEventEmitter } = require('react-native');